analyzer/src/funTest/kotlin/PhpComposerTest.kt

```diff
@@ -65,13 +63,10 @@ class PhpComposerTest : StringSpec() {
         }
     }
 
-    private fun patchExpectedResult(workingDir: File, result: String): String {
-        val vcsPath = workingDir.relativeTo(rootDir).invariantSeparatorsPath
-
-        return result
-                // project.vcs_processed:
-                .replaceFirst("<REPLACE_URL>", normalizeVcsUrl(vcsUrl))
-                .replaceFirst("<REPLACE_REVISION>", vcsRevision)
-                .replaceFirst("<REPLACE_PATH>", vcsPath)
-    }
+    private fun patchExpectedResult(projectDir: File, result: String) =
```

Comment:
I think we have a few identical versions of this method. Can we move that to a base class?

Reply:
That can be done in a follow-up PR of course. Approving as is.

Reply:
Or a utility function, yes. I was also thinking of that. But not as part of this PR 😉